src/components/ActivityLog/ActivityHighlightsContent.tsx

```diff
@@ -98,6 +119,20 @@ const ActivityHighlightsContent = ({
           const year = latestReport.endDate.split('-')[0];
           const latestReportQuarter = `${latestReport.quarter} ${year}`;
 
+          // only use it if it exists in the dropdown options
+          if (dropdownOptions.some((option) => option.value === latestReportQuarter)) {
+            defaultQuarter = latestReportQuarter;
+          }
+        }
+      } else {
+        if (publishedReports && (publishedReports?.length || 0) > 0) {
+          const sorted = [...publishedReports].sort((a, b) =>
+            b.endDate.localeCompare(a.endDate, activeLocale || undefined)
+          );
+          const latestReport = sorted[0];
+          const year = latestReport.endDate.split('-')[0];
+          const latestReportQuarter = `${latestReport.quarter} ${year}`;
```

Comment:
Might be worth creating a function that does this, since both `if` blocks use the same sorting and quarter retrieval